.github/workflows/build-test-vm.yaml

```diff
@@ -22,10 +22,15 @@ on:
         type: boolean
         required: false
         default: false
+      archs:
+        description: 'Architecture to build for'
+        type: string
+        required: false
+        default: '["amd64","arm64"]'
```

Comment:
question: Should these match the format of `build-images.yaml`

Reply:
fixed